FLEDGE.md

```diff
@@ -224,7 +228,8 @@ Once the bids are known, the seller runs code inside an _auction worklet_.  With
 
 
 ```
-scoreAd(adMetadata, bid, auctionConfig, trustedScoringSignals, browserSignals) {
+scoreAd(adMetadata, bid, auctionConfig, trustedScoringSignals, browserSignals,
+extraSellerSignals) {
```

Comment:
What are extraSellerSignals?  Are these extra signals for https://seller.com, or for https://buyer.com, where buyer.com is the buyer that generated the bid?

Reply:
These are the extra signals for https://seller.com -- I'll update the note.